Formula/moarvm.rb

```diff
@@ -1,8 +1,9 @@
 class Moarvm < Formula
   desc "Virtual machine for NQP and Rakudo Perl 6"
   homepage "https://moarvm.org"
-  url "https://github.com/MoarVM/MoarVM/releases/download/2020.09/MoarVM-2020.09.tar.gz"
-  sha256 "48b21922f8843d53a3bf4c04ef357f1d9d2d8cd96f63c5eb59df72eceda6f922"
+  # version usually matches nqp & rakudo
```

Comment:
Not sure if this should have been phrased as a suggestion or a mandate. But maybe if we want the versions between these formulae to always be in sync then it should be an audit check?

Reply:
We have a comment that says the following on the [`git`](https://github.com/Homebrew/homebrew-core/blob/a948d97f0193a49854356fa24bfffe87f6015901/Formula/git.rb#L4) and [`git-gui`](https://github.com/Homebrew/homebrew-core/blob/a948d97f0193a49854356fa24bfffe87f6015901/Formula/git-gui.rb#L4) formulae:

```ruby
# Note: Please keep these values in sync with git-gui.rb when updating.
```

Maybe use that language? I'm not opposed to what you have here but it might be good to be consistent.

I'd be 👍 on an audit for this if it occurs often enough that it's worth implementing. If it's just `git`/`git-gui` and these three, a comment may be sufficient. Also, not sure if this would require modifying the formula DSL at all to indicate shared versions.